src/xdp-request.h

```diff
@@ -101,4 +91,6 @@ auto_lock_helper (GMutex *mutex)
   return mutex;
 }
 
-#define REQUEST_AUTOLOCK(request) G_GNUC_UNUSED __attribute__((cleanup (auto_unlock_helper))) GMutex * G_PASTE (request_auto_unlock, __LINE__) = auto_lock_helper (&request->mutex);
+#define REQUEST_AUTOLOCK(request) \
+  G_GNUC_UNUSED __attribute__((cleanup (auto_unlock_helper))) \
```

Comment:
Another cleanup you can do: there's actually a helper for this in glib: https://docs.gtk.org/glib/func.MUTEX_AUTO_LOCK.html